instance/models/database_server.py

```diff
@@ -120,6 +121,10 @@ class DatabaseServer(ValidateModelMixin, TimeStampedModel):
     class Meta:
         abstract = True
 
+    # Human readable identifier for database instances
+    name = models.CharField(max_length=128, unique=True)
+    description = models.CharField(max_length=128, blank=True)
```

Comment:
@bdero Since we already have similar names on other server models, it would be good to use the opportunity to standardize it a bit - for example, have the same lengths, non-empty requirements, etc. Maybe these fields could go on a `NamedHostMixin` or maybe `NamedDomainMixin`? See for example:
* https://github.com/open-craft/opencraft/blob/08995ce31953f5b140b2b09063617b54cd5c1341/instance/models/appserver.py#L161
* https://github.com/open-craft/opencraft/blob/08995ce31953f5b140b2b09063617b54cd5c1341/instance/models/instance.py#L59

It's less related to the work at hand, so feel free to ignore this, but while we look at the standard mixins, that could also be a good opportunity to group classes which should be inherited by all our models on a single base class - we often forget these when creating new models; ie put `ValidateModelMixin` and `TimeStampedModel` on a `BaseModel`?